ocean_lib/web3_internal/contract_base.py

```diff
@@ -174,12 +171,11 @@ def subscribe_to_event(
 
     @staticmethod
     @enforce_types
-    def get_gas_price(web3) -> int:
+    def get_gas_price(w3: Web3) -> int:
```

Comment:
please use the web3 naming, not w3, we sometimes search through the codebase and it helps to have the same variable name

Reply:
I let `web3` in the first place, but it couldn't be distinguished between the web3 package from the import, so it creates a confusion locally. Can I rename to `web3_object`?